handler/actions/src/main/java/io/knotx/fragments/handler/action/http/options/HttpActionOptions.java

```diff
@@ -70,7 +70,7 @@ public WebClientOptions getWebClientOptions() {
    * @param webClientOptions {@link WebClientOptions} object
    * @return a reference to this, so the API can be used fluently
    */
-  HttpActionOptions setWebClientOptions(WebClientOptions webClientOptions) {
+  public HttpActionOptions setWebClientOptions(WebClientOptions webClientOptions) {
```

Comment:
! This change reverts a bug that is currently present on `master` branch.
Without `public` modifier, the configuration entries are not processed by a converter - which results in using default options.